modules/ztp-image-based-upgrade.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Image-based upgrade for {sno} cluster with the {lcao}
+
+You can upgrade the platform version of a managed {sno} cluster with the {lcao}.
+Using a generated seed image allows you to skip up to two minor releases and it is faster than the standard upgrade method.
+
+You generate the seed image from a {sno} seed cluster, which is deployed and configured with the desired {product-title} and applications.
+You can then use that seed image to upgrade the platform version on any {sno} that has the same combination of hardware, Day 2 Operators, and cluster configuration as the seed cluster.
+
+[IMPORTANT]
+====
+The image-based upgrade uses custom images that are specific to the hardware platform that the clusters are running on.
+Each different hardware platform requires a separate seed image.
+====
+// TODO agree on how much we should specify this. Is this enough or detail about CPU topology, deployment method, etc.
+
+The {lcao} uses two custom resources (CRs) on the participating clusters:
+
+* On the seed cluster, the `SeedGenerator` CR allows the seed image generation. This CR specifies the repository to push the seed image to.
+* On the target cluster, the `ImageBasedUpgrade` CR specifies the seed container image for the upgrade of the target cluster and the backup configurations for your workloads.
+
+.Example SeedGenerator CR
+[source,yaml]
+----
+apiVersion: lca.openshift.io/v1alpha1
+kind: SeedGenerator
+metadata:
+  name: seedimage
+spec:
+  seedImage: <seed_container_image>
+----
+
+.Example ImageBasedUpgrade CR
+[source,yaml]
+----
+apiVersion: lca.openshift.io/v1alpha1
+kind: ImageBasedUpgrade
+metadata:
+  generation: 1
+  name: upgrade
+spec:
+  stage: Idle
+status:
+  conditions:
+  - lastTransitionTime: "2024-01-19T05:42:58Z"
+    message: Idle
+    observedGeneration: 1
+    reason: Idle
+    status: "True"
+    type: Idle
+  observedGeneration: 1
+----
+
+After generating the seed image on the seed cluster, you can move through the stages on the target cluster by specifying the image to pull and then setting the `spec.stage` field to the following values in the `ImageBasedUpgrade` CR:
+
+* `Idle`
+* `Prep`
+* `Upgrade`
+* `Rollback` (Optional)
+* `Idle`
```

Comment:
`Idle` is listed twice?

Reply:
There was a few iterations to this because you start in Idle and end the process in Idle. I'll delete the extra one and see if I can add a diagram to demonstrate the flow better.